invite/src/setup_db.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020, the AMP HTML authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {InviteAction} from './types';
+import {Database, dbConnect} from './db';
+
+/** Set up the database table. */
+export function setupDb(db: Database) {
+  return db.schema.createTable('invites', table => {
+    table.increments('id').primary();
+    table.string('username', 40);
+    table.string('repo', 100);
+    table.integer('issue_number');
+    table.enum('action', Object.values(InviteAction));
+    table.boolean('archived').defaultTo(false);
```

Comment:
Defaults that are not `null` are okay as fallbacks but make sure the code explicitly sets when you first create a row :)

Reply:
Ack